projects/vdk-core/src/vdk/internal/core/errors.py

```diff
@@ -107,6 +107,17 @@ class UserCodeError(DomainError):
     pass
 
 
+class CancelJobExecutionException(DomainError):
```

Comment:
We have an API for canceling job execution. as this is not doing that, it would confuse users. 

Better names I think are "cancel remaining steps" or "skip remaining steps"

Reply:
Did you forget to rename it ?